elastic/test_elastic.py

```diff
@@ -41,6 +41,8 @@
     "elasticsearch.primaries.search.fetch.current": ("gauge", "_all.primaries.search.fetch_current")
 }
 
+PRIMARY_SHARD_INDEX_COUNT = "elasticsearch.indices.count"
```

Comment:
I think that for consistency, I would rather have this in one of the metric dicts (or a new one)... From the source down below it looks like the metric _should_ be available across all versions as you don't use any conditional to include the metric or not. So you might as well just include it in `PRIMARY_SHARD_METRICS`.